Esterni/Piano_qualifica/res/sections/test/integrazione.tex

```diff
@@ -0,0 +1,18 @@
+\subsection{Test di Integrazione}
+
+	\subsubsection{Specifica}
+		Il test di integrazione ha l'obiettivo di verificare la correttezza funzionale nell’interazione tra più moduli\ped{\textit{G}}. In particolare questo tipo di test verifica:
```

Comment:
@luben5691 forse sarebbe piú corretto parlare di test di integrazione al plurale? dato che sono multipli i test di questo tipo  (vale anche per i paragrafi successivi test di regressione ecc..)

Reply:
Si, corretto. Ora dovrebbe essere definito meglio. Prova a darci un occhio e se va bene procedi pure con il merge in develop.
Grazie ;-)